cmd/core/main.go

```diff
@@ -95,6 +96,7 @@ func main() {
 	flag.BoolVar(&controllerArgs.ApplyOnceOnly, "apply-once-only", false,
 		"For the purpose of some production environment that workload or trait should not be affected if no spec change")
 	flag.StringVar(&disableCaps, "disable-caps", "", "To be disabled builtin capability list.")
+	flag.StringVar(&storageDriver, "storage-driver", "Local", "Application file save to the storage driver")
```

Comment:
please use the const variable `driver.LocalDriverName` here instead of `Local`

Reply:
Done